db/migrate/20161101183239_add_active_to_repo.rb

```diff
@@ -0,0 +1,5 @@
+class AddActiveToRepo < ActiveRecord::Migration[5.0]
+  def change
+    add_column :repos, :active, :boolean
```

Comment:
You're leaving yourself open for the three-state boolean (`true`, `false` and `nil`). You should add `null: false, default: false` to protect yourself.